doc/sphinx-guides/source/developers/dev-environment.rst

```diff
@@ -36,29 +36,29 @@ Install Netbeans or Maven
 
 NetBeans IDE is recommended, and can be downloaded from http://netbeans.org . Developers may use any editor or IDE. We recommend NetBeans because it is free, works cross platform, has good support for Jakarta EE projects, and includes a required build tool, Maven.
 
-Below we describe how to build the Dataverse war file with Netbeans but if you prefer to use only Maven, you can find installation instructions in the :doc:`tools` section.
+Below we describe how to build the Dataverse software war file with Netbeans but if you prefer to use only Maven, you can find installation instructions in the :doc:`tools` section.
 
 Install Homebrew (Mac Only)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 On Mac, install Homebrew to simplify the steps below: https://brew.sh
 
-Clone the Dataverse Git Repo
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Clone the Dataverse Software Git Repo
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Fork https://github.com/IQSS/dataverse and then clone your fork like this:
 
 ``git clone git@github.com:[YOUR GITHUB USERNAME]/dataverse.git``
 
-Build the Dataverse War File
+Build the Dataverse Software War File
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Formatting requires additional header characters for this line to match the header text length above.

Reply:
fixed in https://github.com/IQSS/dataverse/pull/7567/commits/f41355f317062479dce6cb4d02f716459681e8cc